libs/design/src/atoms/button/button.component.scss

```diff
@@ -11,7 +11,7 @@
 	position: relative;
 	text-align: center;
 	text-decoration: none;
-	transition: all 0.3s ease; //sass-lint:disable-line no-transition-all
+	transition: all 300ms ease; //sass-lint:disable-line no-transition-all
```

Comment:
Sass-lint disable comments could be removed.

Reply:
fixed in c21085796e7d54de741ee3ccb5c1c78e7d35f18d